src/mavm.rs

```diff
@@ -277,33 +278,41 @@ impl fmt::Display for CodePt {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub enum Value {
     Int(Uint256),
-    Tuple(Rc<Vec<Value>>),
+    Tuple(Rc<Vec<Value>>, Uint256),
```

Comment:
This change causes the JSON serialization format to be changed, which I don't think we want. This will probably require changing the serialization format of the `Tuple` type

Reply:
Switched back to the old structure. This is viable because of the change to the ArbOS stack library.